CONTRIBUTING.md

```diff
@@ -16,7 +16,20 @@ To set up a development environment, please follow these steps:
    git clone https://github.com/liam-hq/liam
    ```
 
-2. Install npm package
+2. Run the setup script
+
+   ```sh
+   ./scripts/setup-local-dev.sh
```

Comment:
If we decide to make `./scripts/setup-local-dev.sh` the official entry point for development, it might be a good idea to run CI on macOS (or similar) via GitHub Actions as well.